libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/TelemetryEx.kt

```diff
@@ -28,3 +30,20 @@ internal fun Location.toTelemetryLocation(): TelemetryLocation {
         }
     )
 }
+
+/**
+ * Register a [UserFeedbackObserver] to be notified when a new user feedback is posted.
+ *
+ * @param userFeedbackObserver UserFeedbackObserver
+ */
+fun registerUserFeedbackObserver(userFeedbackObserver: UserFeedbackObserver) {
+    MapboxNavigationTelemetry.registerUserFeedbackObserver(userFeedbackObserver)
+}
```

Comment:
If this is in the internal package then let's move all of the elements to the internal package (including `UserFeedback` and the extensions to `MapboxNavigation` with `postUserFeedback` overloads).

Reply:
What is the point of moving it to the internal package if we have to maintain binary compatibility anyway?

Reply:
Sorry, I wasn't aware of that context :+1:

Reply:
Moved everything to the internal package.